hypertrace-core-graphql-log-event-schema/src/main/java/org/hypertrace/core/graphql/log/event/request/LogEventRequest.java

```diff
@@ -0,0 +1,7 @@
+package org.hypertrace.core.graphql.log.event.request;
+
+import org.hypertrace.core.graphql.context.GraphQlRequestContext;
+
+public interface LogEventRequest {
+  GraphQlRequestContext requestContext();
```

Comment:
ResultSetRequest and ResultSetRequestBuilder may be reusable for this feature (although I suspect they'd force the issue on requiring a time range)

Reply:
Is having a separate RequestBuilder problematic (wouldn't it better for extension in future)?